README.md

```diff
@@ -1,19 +1,17 @@
-# portfolio-project
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# portfolio-project-mobile-menu
+![](https://github.com/maherramadan78/Myportfolio/tree/desktop-version)
 
 # Myporfolio
 
-> first mobile portfolio contact form
-
-## html and styles css
+## html and styles css & javascript
 
 - english language
 - visual code editor
 - html , css.
 
```

Comment:
- [ ] Microverse released a new [README template](https://github.com/microverseinc/readme-template). I suggest you update it. Your project is almost complete, keep it up! 🚀

Reply:
thank you